packages/grafana-ui/src/components/InfoBox/InfoBox.tsx

```diff
@@ -46,7 +46,7 @@ export const InfoBox = React.memo(
           </div>
           <div>{children}</div>
           {url && (
-            <a href={url} className={styles.docsLink} target="_blank">
+            <a href={url} className={styles.docsLink} target="_blank" rel="noreferrer">
```

Comment:
Wonder if it would be a good idea to make a component for this, e.g. `ExternalLink`, that would'd have `target` and `rel` predefined.